### ISSUE TITLE
Broken after the release of Season of the Splicer.

### ISSUE THREAD

**Issue by JavikN7** [ElbowBaggins/GinsorAudioTool2Plus#7]
You are amazing for supporting and iterating on this tool, thank you for doing that. Bungie changed something when they released this season, and now when trying to play any voiceline after rebuilding the DB it says "No Audiofile for selected Transcript." in the console. 

what do
![image_2021-05-12_174218](https://user-images.githubusercontent.com/69909068/117994706-6695a600-b349-11eb-8c1e-936907bc148e.png)

**Comment by RumekFuria** [comment]
same here.
so yeah, when you can, look into this Baggins

**Comment by dominicmarten** [comment]
Yeah. Seems like most of the entries are "missing" the soundfiles. So, when you get a chance, can you look into it? I'll buy you a coffee 😉

**Comment by ElbowBaggins** [comment]
This is probably happening due to fragility inherent in Ginsor's design. I have encountered this behavior before (around BL release) and, essentially, some identifying marker indicating that a file is "voice lines audio" has (probably) changed. I suspect this will be a minor fix, it's just a matter of finding where it is.

**Comment by thejhobe** [comment]
Any ETA? I just found this tool and was trying it out. Only to find it's broken :(

**Comment by RumekFuria** [comment]
> This is probably happening due to fragility inherent in Ginsor's design. I have encountered this behavior before (around BL release) and, essentially, some identifying marker indicating that a file is "voice lines audio" has (probably) changed. I suspect this will be a minor fix, it's just a matter of finding where it is.


>"A minor fix"
>Entire season passes and we have the next one already
>no minor fix yet

**Comment by ElbowBaggins** [comment]
Issue is duplicated by #8, #9, and #10. For reference.
